<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpanTreeClass</class>
 <widget class="QMainWindow" name="SpanTreeClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpanTree</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Row Count</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="rowSpinBox"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Column Count</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="colSpinBox"/>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> Order</string>
       </property>
      </widget>
      <widget class="QComboBox" name="orderComboBox"/>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="stCheckBox">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Show Spanning Tree</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QWidget" name="canvasWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SpanTree.qrc"/>
 </resources>
 <connections/>
</ui>
